packages/plugin-default-event-tracking-advanced-browser/src/helpers.ts

```diff
@@ -57,6 +58,39 @@ export const getText = (element: Element): string => {
   return text;
 };
 
+export const getSelector = (element: Element, logger?: Logger): string => {
+  let selector = '';
+  try {
+    selector = finder(element, {
+      className: (name: string) => name !== constants.AMPLITUDE_VISUAL_TAGGING_HIGHLIGHT_CLASS,
+    });
+    return selector;
+  } catch (error) {
+    if (logger && logger.warn) {
```

Comment:
Why the check for logger.warn here?

Reply:
Thanks for reviewing! Safe to remove, updated!